.github/workflows/selenium_beta.yaml

```
@@ -0,0 +1,51 @@
+name: Selenium tests (beta history panel)
+on: [push, pull_request]
+env:
+  GALAXY_TEST_DBURI: 'postgresql://postgres:postgres@localhost:5432/galaxy?client_encoding=utf8'
+  GALAXY_TEST_SKIP_FLAKEY_TESTS_ON_ERROR: 'true'
```

```suggestion
GALAXY_TEST_SKIP_FLAKEY_TESTS_ON_ERROR: 1
```
not a big deal, just a little rant for consistency.
 Was just wandering why `GALAXY_TEST_SELENIUM_BETA_HISTORY=0`, `GALAXY_TEST_SELENIUM_HEADLESS=0` and `GALAXY_SKIP_CLIENT_BUILD=1` are written in tutorial to be equal 1 or 0 and GALAXY_TEST_SKIP_FLAKEY_TESTS_ON_ERROR is suddenly `true`

---

Done.